### ISSUE TITLE
Include response file

### ISSUE THREAD

**Issue by AronRubin** [15knots/cmake4eclipse#47]
Note I have to use the following to have the compile_command.json parser work as it is unable to follow the reference.

set(CMAKE_CXX_USE_RESPONSE_FILE_FOR_INCLUDES 0)

Example compile_commands.json without the above directive:
```
[
{
  "directory": "C:/Users/arubin/workspace/FaceService/build/Debug/src",
  "command": "/C/msys64/mingw64/bin/g++.exe   @CMakeFiles/FaceService.dir/includes_CXX.rsp  -std=c++14 -g   -std=c++11 -o CMakeFiles/FaceService.dir/main
.cpp.obj -c /C/Users/arubin/workspace/FaceService/src/main.cpp",
  "file": "C:/Users/arubin/workspace/FaceService/src/main.cpp"
}
]
```
here is the corresponding includes_CXX.rsp:
```
-I"C:/Program Files/dlib/include"
```

**Comment by 15knots** [comment]
Thank you for reporting a solution!
I already noticed that issue with the compile_command.json parser on windows but found no solution except writing lots of parsers for whatever the compiler`s response file syntax is.

Maybe, I should add a *Tip and Tricks' section to the plugin help that mentions the `CMAKE_${lang}_USE_RESPONSE_FILE_FOR_INCLUDES 1` variable...

**Comment by 15knots** [comment]
fixed in c98219f28d5c13253cb1ca95a016644353ce8687
